pint/src/main/resources/standards/pint/instructions/prompt-close-transfer.md

```diff
@@ -0,0 +1,42 @@
+This action completes an electronic Bill of Lading (eBL) transfer that was previously initiated in a PINT (Platform
+Interoperability) scenario.
+
+**What your sending platform should do:**
+
+1. **Retrieve the envelope reference:** Use the envelopeReference from the previous transfer initiation
+2. **Send finish request:** Make a PUT request to the receiving platform's finish-transfer endpoint
+3. **Complete the transfer:** This formally closes the transfer process
+
+**Request details:**
+
+- **HTTP Method:** PUT
+- **Endpoint:** `/v{version}/envelopes/{envelopeReference}/finish-transfer`
+- **Request Body:** Empty (no payload required)
+- **Headers:** Standard API version and authentication headers
+
+**What this action represents:**
+
+- The final step in a multistep eBL transfer process
+- Signals to the receiving platform that the transfer should be completed
+- Confirms that all required documents and information have been provided
+
+**Expected response:**
+
+- Variable HTTP status code based on the scenario being tested
+- **Signed response payload** confirming the transfer completion
+- The receiving platform should finalize and close the transfer
+
+**When this action is used:**
+
+- After successfully initiating a transfer with `PintInitiateTransferAction`
```

Comment:
This seems like an interal code reference - does the Conformance tester know anything about a `PintInitiateTransferAction` ?